### PR TITLE
fix: correctly imports commonjs "debug" package

### PR DESCRIPTION
**Pull request by thebuilder**
Fix for #350 

Changes the imports, so they handle the CommonJS exports from the "debug" package.

### CODE REVIEW

File: src/Interceptor.ts [hunk 1]
```diff
@@ -1,11 +1,14 @@
-import { Debugger, debug } from 'debug'
+import type { Debugger } from 'debug'
+import pkg from 'debug'
```
Comment (kettanaito) [inline src/Interceptor.ts:2]: Perhaps let's name it something else than a generic `pkg`? In fact, maybe we should depict the reason for this kind of import in the name itself:

```js
import debugCjsImport from 'debug'

const debug = debugCjsImport
```

What do you think about this?

**Review by kettanaito — COMMENTED** [review]
The changes are solid, I've but one comment regarding the import name. Would like to know what you think.

File: src/Interceptor.ts (hunk 1 above)
Comment (thebuilder) [reply src/Interceptor.ts:2]: Sure - Makes sense.

**Comment by thebuilder** [comment]
Looking through the code, I see there's a lot more references to `debug` in the code.

Think it might make more sense to make a simple "wrapper" around around it, that essentially re-exports it as ESM. Otherwise we'll end up doing this import dance in a lot of places.

**Comment by kettanaito** [comment]
That's true, we use `debug` quite extensively. Having some util makes sense. I was thinking about something like

```js
// src/utils/debug.ts
import debugCjsImport from 'debug'

export const debug = debugCjsImport
```

What do you have in mind?

**Comment by thebuilder** [comment]
Yeah, exactly like that 👍

**Comment by kettanaito** [comment]
Let me know if this is ready for review 👀

**Comment by thebuilder** [comment]
Should be! Replaced all the debug imports with the one from utils

**Review by kettanaito — APPROVED** [review]
Thank you, @thebuilder! Welcome to contributors 🎉

**Comment by kettanaito** [comment]
## Released: v0.22.5 🎉

This has been released in v0.22.5!

- 📄 [**Release notes**](https://github.com/mswjs/interceptors/releases/tag/v0.22.5)
- 📦 [npm package](https://www.npmjs.com/package/@mswjs/interceptors/v/0.22.5)

Make sure to always update to the latest version (`npm i @mswjs/interceptors@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.

**Pull request merged**